Server/src/migrations/SeedDatabase.ts

```diff
@@ -324,7 +327,7 @@ export class SeedDatabase1611344612000 implements MigrationInterface {
     newGraph.axisUnits = ["C", "mm"];
     await connection.manager.save(newGraph);
 
-    let newGraph2 = new Savedgraphs();
+    let newGraph2 = new Graphstate();

```

Comment:
why no PascalCase? GraphState?

Reply:
Because having multiple capital letters in an Entity results in a funky table name in the DB when the table is generated by TypeORM.